### PR TITLE
Broad, shallow cleanup

### PR DESCRIPTION
**Pull request by software-dov**
Unify include style and ordering of headers
Add gax and gax/include to include paths where appropriate
Add missing namespace close comments
Remove FRIEND_TEST macro from CallContext

### CODE REVIEW

File: gax/call_context_test.cc [hunk 1]
```diff
@@ -12,11 +12,10 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-#include "call_context.h"
-#include "backoff_policy.h"
+#include "gax/call_context.h"
 #include "googletest/include/gtest/gtest.h"
```
Comment (coryan) [inline gax/call_context_test.cc]: Why sometimes `<gtest/gtest.h>` and sometimes `"googletest/include/gtest/gtest.h"`?

File: generator/standalone.cc [hunk 2]
```diff
@@ -1,9 +1,8 @@
-#include <fstream>
-#include <standalone.h>
-
 #include <google/protobuf/compiler/command_line_interface.h>
 #include <google/protobuf/stubs/io_win32.h>
+#include <fstream>
 #include <gapic_generator.h>
```
Comment (coryan) [inline generator/standalone.cc]: Is this part of your project? If so, you probably want quotes?

File: generator/standalone.cc [hunk 3]
```diff
@@ -1,9 +1,8 @@
...
+#include <standalone.h>
```
Comment (coryan) [inline generator/standalone.cc]: Ditto here.

File: generator/standalone.cc (hunk 2 above)
Comment (software-dov) [reply generator/standalone.cc]: Aaaah. I'll fix that right away.

File: gax/call_context_test.cc (hunk 1 above)
Comment (software-dov) [reply gax/call_context_test.cc]: Different people making different PRs. I'll unify the convention.

**Pull request merged**